.github/workflows/memory.yml

```diff
@@ -0,0 +1,37 @@
+name: Memory CI/CD
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-24.04
+    timeout-minutes: 10
+    defaults:
+      run:
+        working-directory: ./typescript/packages/common-memory
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Deno
+        uses: denoland/setup-deno@v2
+        with:
+          deno-version: "2.1.9"
+
+      - name: Cache dependencies
+        uses: actions/cache@v3
+        with:
+          path: |
+            ~/.deno
+            ~/.cache/deno
+          key: ${{ runner.os }}-deno-${{ hashFiles('**/deno.json') }}
+
+      - name: Run tests
+        working-directory: typescript/packages/common-memory
```

Comment:
This should be covered in the `unit-tests.yml` now with the other deno projects

Reply:
Thanks for flagging, I removed that file.